config/StoredProcedures/GetAllInventory.sql

```diff
@@ -30,4 +31,4 @@ on s1.ProductID = s2.ProductID;
 
 END $$
 
-DELIMITER ;
\ No newline at end of file
+DELIMITER ;
```

Comment:
I was hoping to avoid that, but it looks like you're correct. Unfortunately though it looks like your procedure has the same problem. You can tell by running it and looking at the row for product 101, and then comparing those results with the Runs table. I'll see if I can work out a fix later, unless you feel like working on it before then.